fe/fe-core/src/main/java/com/starrocks/common/util/KafkaUtil.java

```diff
@@ -214,7 +214,7 @@ private PProxyResult sendProxyRequest(PProxyRequest request) throws UserExceptio
                     PProxyResult result;
                     try {
                         result = future.get(Config.routine_load_kafka_timeout_second, TimeUnit.SECONDS);
-                    } catch (TimeoutException e) {
+                    } catch (Exception e) {
                         LOG.warn("failed to send proxy request to " + address + " err " + e.getMessage());
                         // When getting kafka info timed out, we tried again three times.
                         if (++retryTimes > 3 || (retryTimes + 1) * Config.routine_load_kafka_timeout_second >
```

Comment:
Try when encountering any Exception?

Reply:
> Retry when encountering any Exception?

yes

Reply:
If is an exception about privilege, it seems no necessary to retry.